code/game/objects/structures/tables_racks.dm

```diff
@@ -28,7 +28,8 @@
 	var/busy = 0
 	var/buildstackamount = 1
 	var/framestackamount = 2
-	var/deconstructable = 1

```

Comment:
The var was only used for abductor's optable, I've replaced it by using NODECONSTRUCT and overriding table_destroy().